### ISSUE TITLE
Add emulation for windres

### ISSUE THREAD

**Issue by tn1997tn** [LADSoft/OrangeC#786]
A wrapper for `orc` to emulate `windres`.

**Comment by GitMensch** [comment]
[windres is](https://man7.org/linux/man-pages/man1/windres.1.html):
`windres [options] [input-file] [output-file]`

[orc is](https://github.com/LADSoft/OrangeC/blob/master/doc/general/ORC.md):
`ORC [options] rcfile.rc`

The main question is: What do you want for emulation?
I _guess_ it is neither support for res files or coff input files, nor conversion to .rc, is it?

I _think_ you likely want something like [gccocc](https://github.com/LADSoft/OrangeC/blob/master/doc/wrappers/gccocc.md) mapping `windresocc -i input-file -I include outout-file` to `orc -i include -o output-file input-file` and the like, right?

In this case: have my vote, in the other case, too :-)

@LADSoft: would supporting res and coff input files be reasonable/possible? What about converting to .rc as windres can?

**Comment by tn1997tn** [comment]
Yes, something like `gccocc`. It's a bit lazy to have to rewrite `windres` command to `orc` but not something I can't do.

**Comment by LADSoft** [comment]
you could open a separate issue to support .RES and .COFF files, however, that would be a VERY low priority compared to all the other stuff we are trying to get done.    I could write a wrapper that that compiles RC files, and invokes the linker (I assume wndres invokes the linker lol)

**Comment by tn1997tn** [comment]
> you could open a separate issue to support .RES and .COFF files, however, that would be a VERY low priority compared to all the other stuff we are trying to get done. I could write a wrapper that that compiles RC files, and invokes the linker (I assume wndres invokes the linker lol)

It's @GitMensch needs support for .RES and .COFF files, not me.

**Comment by iahunq** [comment]
@LADSoft Should this issue be closed?

**Comment by iahunq** [comment]
@LADSoft @GitMensch Should this issue be closed?

**Comment by LADSoft** [comment]
I was thinking I would actually do this one...   as long as we are doing wrappers...
